home-api-tests/src/test/java/NewsApiTest.java

```diff
@@ -0,0 +1,108 @@
+import java.util.HashMap;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import com.softserveinc.ita.homeproject.ApiClient;
+import com.softserveinc.ita.homeproject.ApiException;
+import com.softserveinc.ita.homeproject.ServerConfiguration;
+import com.softserveinc.ita.homeproject.api.NewsApi;
+import com.softserveinc.ita.homeproject.model.CreateNews;
+import com.softserveinc.ita.homeproject.model.ReadNews;
+import com.softserveinc.ita.homeproject.model.UpdateNews;
+
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class NewsApiTest {
+
+    private final NewsApi newsApi;
```

Comment:
may by it's better to use annotation  @BeforeAll instead constructor

Reply:
@BeforeAll is a class-level annotation and will be used before all test runs. It isn't applicable to our needs since we would like to keep our tests as clean as it is possible. But using @Before/@BeforeEach will be OK.